pkg/v1/mutate/mutate.go

```diff
@@ -335,7 +335,7 @@ func layerTime(layer v1.Layer, t time.Time) (v1.Layer, error) {
 	b := w.Bytes()
 	// gzip the contents, then create the layer
 	opener := func() (io.ReadCloser, error) {
-		return v1util.GzipReadCloser(ioutil.NopCloser(bytes.NewReader(b))), nil
+		return gzip.GzipReadCloser(ioutil.NopCloser(bytes.NewReader(b))), nil
```

Comment:
Drop the stutter?

Reply:
What do you want to call the Gunzip variant :)

Reply:
You got me there.

Reply:
I was piggybacking on the sentiment @ImJasonH had below that renaming things becomes *reeeeaaaaal* easy under `internal/`, so this is trivial to fixup later if we find a name we like. 🤷